.changeset/sour-badgers-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-api': patch
+---
+
+Add argument for storage key to GitHub Auth Api for customizing it's value.
```

Comment:
This will need a tweak as well 😅 . Thanks for the quick update!

Reply:
oh yeah, I forgot about the change set